Core/RendererFactory.py

```diff
@@ -3,28 +3,48 @@
 from Graphics.SceneRenderer import SceneRenderer
 
 from Graphics.PostProcessing.Tonemap import Tonemap
+from Graphics.PostProcessing.Bloom import Bloom
 
-class RendererFactory():
 
+class RendererFactory:
     def create_renderer(window):
 
         scene_renderer = SceneRenderer(window)
 
-        #Background color of the scene (replaces the arcade.background thing), is affected by lighting
-        scene_renderer.background_color = (0.5,0.5,0.5,1.0)
+        # Background color of the scene (replaces the arcade.background thing), is affected by lighting
+        scene_renderer.background_color = (0.5, 0.5, 0.5, 1.0)
 
-        #Ambient light value applied to every pixel
+        # Ambient light value applied to every pixel
         scene_renderer.light_renderer.ambient_light = (1.0, 1.0, 1.0)
 
-        #Setup all post processing
+        # Setup all post processing
         RendererFactory.setup_post_processing(scene_renderer)
 
         return scene_renderer
 
+<<<<<<< HEAD
+    def setup_post_processing(scene_renderer):
+=======
     def setup_post_processing(scene_renderer):      
         
+        bloom = Bloom()
+        scene_renderer.post_processing.add_effect(bloom)
+
+        #TEMP
+        bloom.power = 1.5
+        bloom.threshold = 0.75
+
+>>>>>>> LightFalloff
+
         tonemap = Tonemap()
         scene_renderer.post_processing.add_effect(tonemap)
 
+<<<<<<< HEAD
+        # Set what HDR value gets mapped to 100% white on your monitor
+        tonemap.white_point = 2.0
+=======
         #Set what HDR value gets mapped to 100% white on your monitor
-        tonemap.white_point = 2.0
\ No newline at end of file
+        tonemap.white_point = 3.0
+
+    
+>>>>>>> LightFalloff
```

Comment:
Code has merge markers in it. This might be because I just dropped my PR on the repo before reviewing this.